### PR TITLE
docs(api): expand and improve Deck Slots page

### PR DESCRIPTION
**Pull request by ecormany**
# Overview

This PR builds upon the initial stub for the Deck Slots page in the Python API docs. It adds an SVG deck map image for comparison between Flex and OT-2, as well as a text-based lookup table. It also generally fills out the prose and adds a TODO at the bottom for future description of staging slots in PAPI.

# Test Plan

Check out the [sandbox](http://sandbox.docs.opentrons.com/docs-deck-slots-destub/v2/deck_slots.html), especially making sure that the image behaves well.

# Changelog

- reorg into two sections
- replace tabular deck maps with SVG deck maps
- add table of deck slot equivalences
- TODO for future staging slots
- general cleanup

# Review requests

Nothing special.

# Risk assessment

Nil, docs.

### CODE REVIEW

File: api/docs/v2/deck_slots.rst [hunk 1]
```diff
@@ -7,68 +7,79 @@
 
 .. _deck-slots:
 
-##########
+**********
```
Comment (SyntaxColoring) [inline api/docs/v2/deck_slots.rst:10]: For my reference, are our .rst heading conventions written down anywhere?

**Review by SyntaxColoring — APPROVED** [review]
Très cool. Très factually correct.

File: api/docs/v2/deck_slots.rst (hunk 1 above)
Comment (ecormany) [reply api/docs/v2/deck_slots.rst:10]: Yes, will send you an internal link with the info.

File: api/docs/v2/deck_slots.rst [hunk 2]
```diff
@@ -7,68 +7,79 @@
...
 Deck Slots
-##########
+**********
 
-When you load an item onto the robot's deck, like with :py:obj:`ProtocolContext.load_labware()` or :py:obj:`ProtocolContext.load_module()`, you need to specify which slot to put it in.
+When you load an item onto the robot's deck, like with :py:obj:`ProtocolContext.load_labware()` or :py:obj:`ProtocolContext.load_module()`, you need to specify which slot to put it in. The API accepts values that correspond to the physical deck slot labels on an OT-2 or Flex robot.
```
Comment (jwwojak) [inline api/docs/v2/deck_slots.rst:14]: @ecormany That last sentence, flip the robot order, like this:

> . . . deck slot labels on Flex or an OT-2.

I see this is already merged. I can make the change in one of the open PRs that I'm working on.

**Pull request merged**